semantic-conventions/src/opentelemetry/semconv/model/semantic_convention.py

```diff
@@ -264,6 +264,9 @@ def __init__(self, group):
         self.metric_name = group.get("metric_name")
         self.unit = group.get("unit")
         self.instrument = group.get("instrument")
+
+        namespaces = self.metric_name.split(".")
+        self.root_namespace = namespaces[0] if len(namespaces) > 1 else ""
```

Comment:
in this example we have root namespace only if "." is present in name?

Reply:
yes, otherwise there's no namespace at all but only the attribute / metric name